docs/data/material/getting-started/templates/checkout/getCheckoutTheme.tsx

```diff
@@ -0,0 +1,620 @@
+      fontSize: 18,
+    },
+    subtitle2: {
+      fontSize: 16,
+    },
+    body1: {
+      fontWeight: 400,
+      fontSize: 15,
+    },
+    body2: {
+      fontWeight: 400,
+      fontSize: 14,
+    },
+    caption: {
+      fontWeight: 400,
+      fontSize: 12,
+    },
+  },
+});
+
+export default function getCheckoutTheme(mode: PaletteMode): ThemeOptions {
+  return {
+    ...getDesignTokens(mode),
+    components: {
+      MuiToggleButtonGroup: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            borderRadius: '10px',
+            boxShadow: `0 4px 16px ${alpha(gray[400], 0.2)}`,
+            '& .Mui-selected': {
+              color: brand[500],
+            },
+            ...(theme.palette.mode === 'dark' && {
+              '& .Mui-selected': {
+                color: '#fff',
+              },
+              boxShadow: `0 4px 16px ${alpha(brand[700], 0.5)}`,
+            }),
+          }),
+        },
+      },
+      MuiToggleButton: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            padding: '12px 16px',
+            textTransform: 'none',
+            borderRadius: '10px',
+            fontWeight: 500,
+            ...(theme.palette.mode === 'dark' && {
+              color: gray[400],
+              boxShadow: '0 4px 16px rgba(0, 0, 0, 0.5)',
+              '&.Mui-selected': { color: brand[300] },
+            }),
+          }),
+        },
+      },
+      MuiButtonBase: {
+        defaultProps: {
+          disableTouchRipple: true,
+          disableRipple: true,
+        },
+        styleOverrides: {
+          root: {
+            boxSizing: 'border-box',
+            transition: 'all 100ms ease-in',
+            '&:focus-visible': {
+              outline: `3px solid ${alpha(brand[500], 0.5)}`,
+              outlineOffset: '2px',
+            },
+          },
+        },
+      },
+      MuiButton: {
+        styleOverrides: {
+          root: ({ theme, ownerState }) => ({
+            boxSizing: 'border-box',
+            boxShadow: 'none',
+            borderRadius: '10px',
+            textTransform: 'none',
+            '&:active': {
+              transform: 'scale(0.98)',
+            },
+            ...(ownerState.size === 'small' && {
+              maxHeight: '32px',
+            }),
+            ...(ownerState.size === 'medium' && {
+              height: '40px',
+            }),
+            ...(ownerState.variant === 'contained' &&
+              ownerState.color === 'primary' && {
+                color: brand[50],
+                background: brand[500],
+                backgroundImage: `linear-gradient(to bottom, ${brand[400]}, ${brand[600]})`,
+                boxShadow: `inset 0 1px ${alpha(brand[300], 0.4)}`,
+                border: `1px solid ${brand[500]}`,
+                '&:hover': {
+                  background: brand[400],
+                  backgroundImage: 'none',
+                  boxShadow: `0 0 0 1px  ${alpha(brand[300], 0.5)}`,
```

Comment:
```suggestion
                backgroundColor: brand[500],
                backgroundImage: `linear-gradient(to bottom, ${brand[400]}, ${brand[500]})`,
                boxShadow: `inset 0 1px ${alpha(brand[300], 0.5)}, inset 0 -2px ${alpha(brand[700], 0.5)}`,
                border: `1px solid ${brand[500]}`,
                '&:hover': {
                  backgroundColor: brand[400],
                  backgroundImage: 'none',
                  boxShadow: `0 0 0 1px  ${alpha(brand[300], 0.5)}`,
```

Fine-tuning the button styles a bit